stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/composer/MessageComposerViewModelBinding.kt

```diff
@@ -1,80 +1,30 @@
 package io.getstream.chat.android.ui.message.composer
 
-import android.text.Editable
-import android.view.ViewGroup
-import androidx.core.view.isVisible
 import androidx.lifecycle.LifecycleOwner
-import androidx.lifecycle.viewModelScope
-import com.google.android.material.internal.TextWatcherAdapter
-import io.getstream.chat.android.ui.common.extensions.internal.streamThemeInflater
-import io.getstream.chat.android.ui.databinding.StreamUiMessageComposerDefaultCenterContentBinding
-import io.getstream.chat.android.ui.databinding.StreamUiMessageComposerDefaultLeadingContentBinding
-import io.getstream.chat.android.ui.databinding.StreamUiMessageComposerDefaultTrailingContentBinding
+import androidx.lifecycle.lifecycleScope
 import kotlinx.coroutines.flow.collect
 import kotlinx.coroutines.launch
 
-public fun MessageComposerViewModel.bindView(view: MessageComposerView, lifecycleOwner: LifecycleOwner) {
-    view.leadingContent = { container: ViewGroup ->
-        val binding = StreamUiMessageComposerDefaultLeadingContentBinding.inflate(
-            view.streamThemeInflater,
-            container,
-            false
-        )
-        binding.apply {
-            attachmentsButton.isVisible = false
-            commandsButton.isVisible = false
-        }.root
+public fun MessageComposerViewModel.bindView(
+    view: MessageComposerView,
+    lifecycleOwner: LifecycleOwner,
+) {
+    view.onSendMessage = {
```

Comment:
We still don't allow the user to customize these actions if they want, we just override them

Reply:
Added possibility for customizing them in `bindView` function's params